pkg/ingress/controller/openstack/octavia.go

```diff
@@ -367,7 +367,11 @@ func (os *OpenStack) EnsureListener(name string, lbID string, secretRefs []strin
 	} else {
 		if len(listenerAllowedCIDRs) > 0 && !reflect.DeepEqual(listener.AllowedCIDRs, listenerAllowedCIDRs) {
 			_, err := listeners.Update(os.Octavia, listener.ID, listeners.UpdateOpts{
-				AllowedCIDRs: &listenerAllowedCIDRs,
+				AllowedCIDRs:         &listenerAllowedCIDRs,
```

Comment:
Shouldn't the timeouts be updated when there is a change in the timeouts and not when allowedCIDRs has changed?

Reply:
@MaysaMacedo: Whoops, I missed this, obviously valid comment. It's fixed in #2518.